### ISSUE TITLE
Update web/server/vue-cli/package-lock.json

### ISSUE THREAD

**Issue by carlsmedstad** [Ericsson/codechecker#3967]
**Describe the bug**

I'm maintainer for the [CodeChecker package](https://aur.archlinux.org/packages/codechecker) in the Arch Linux User repositories.

The web UI is not buildable with a recent version of Node.js (v20.4.0):

```
cd /tmp/makepkg/codechecker/src/codechecker-6.22.2/web/server//vue-cli && \
npm install && \
npm run-script build && \
echo  > /tmp/makepkg/codechecker/src/codechecker-6.22.2/web/server//vue-cli/dist/.build-commit

up to date, audited 1373 packages in 3s

130 packages are looking for funding
  run `npm fund` for details

38 vulnerabilities (12 moderate, 17 high, 9 critical)

To address issues that do not require attention, run:
  npm audit fix

To address all issues (including breaking changes), run:
  npm audit fix --force

Run `npm audit` for details.

> codechecker@1.0.0 build
> rimraf dist && cross-env NODE_ENV=production webpack --config config/webpack.prod.js

Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
  Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating
assets by status 6.95 MiB [cached] 88 assets
Entrypoint main = runtime.db20ca79dc8365955d8a.js vendors.6dc238e2dde0d340d9e4.js main.4f149c319727f5db7d64.js
orphan modules 2.4 MiB [orphan] 1405 modules
runtime modules 7.67 KiB 12 modules
built modules 8.14 MiB [built]
  modules by path ./node_modules/ 6.94 MiB 868 modules
  modules by path ./src/ 1.2 MiB
    modules by path ./src/components/ 692 KiB 95 modules
    modules by path ./src/views/ 392 KiB 19 modules
    modules by path ./src/assets/ 38.6 KiB 15 modules
    modules by path ./src/store/ 19.2 KiB 3 modules
    modules by path ./src/*.vue 1.45 KiB
      ./node_modules/vue-style-loader/index.js!./node_modules/css-loader/dist/cjs.js??clonedRuleSet-4[0].rules[0].use[1]!./node_modules/vue-loader/lib/loaders/stylePostLoader.js!./node_modules/sass-loader/dist/cjs.js??clonedRuleSet-4[0].rules[0].use[2]!./node_modules/vue-loader/lib/index.js??vue-loader-options!./src/App.vue?vue&type=style&index=0&lang=scss& 762 bytes [built] [code generated]
      ./node_modules/css-loader/dist/cjs.js??clonedRuleSet-4[0].rules[0].use[1]!./node_modules/vue-loader/lib/loaders/stylePostLoader.js!./node_modules/sass-loader/dist/cjs.js??clonedRuleSet-4[0].rules[0].use[2]!./node_modules/vue-loader/lib/index.js??vue-loader-options!./src/App.vue?vue&type=style&index=0&lang=scss& 727 bytes [built] [code generated]
    modules by path ./src/mixins/ 8.86 KiB
      ./src/mixins/index.js + 7 modules 7.88 KiB [built] [code generated]
      ./src/mixins/date.mixin.js 1000 bytes [built] [code generated]

ERROR in ./src/assets/logo.png
Module build failed (from ./node_modules/file-loader/dist/cjs.js):
Error: error:0308010C:digital envelope routines::unsupported
    at new Hash (node:internal/crypto/hash:69:19)
    at Object.createHash (node:crypto:138:10)
    at getHashDigest (/tmp/makepkg/codechecker/src/codechecker-6.22.2/web/server/vue-cli/node_modules/file-loader/node_modules/loader-utils/lib/getHashDigest.js:46:34)
    at /tmp/makepkg/codechecker/src/codechecker-6.22.2/web/server/vue-cli/node_modules/file-loader/node_modules/loader-utils/lib/interpolateName.js:113:11
    at String.replace (<anonymous>)
    at interpolateName (/tmp/makepkg/codechecker/src/codechecker-6.22.2/web/server/vue-cli/node_modules/file-loader/node_modules/loader-utils/lib/interpolateName.js:110:8)
    at Object.loader (/tmp/makepkg/codechecker/src/codechecker-6.22.2/web/server/vue-cli/node_modules/file-loader/dist/index.js:29:48)
 @ ./node_modules/vue-loader/lib/loaders/templateLoader.js??vue-loader-options!./node_modules/vue-loader/lib/index.js??vue-loader-options!./src/components/Layout/TheHeader.vue?vue&type=template&id=be44d0c0& 1:772-800
 @ ./src/components/Layout/TheHeader.vue?vue&type=template&id=be44d0c0& 1:0-213 1:0-213
 @ ./src/components/Layout/TheHeader.vue 1:0-88 10:2-8 11:2-17
 @ ./src/components/Layout/index.js 1:0-36 2:0-21
 @ ./node_modules/babel-loader/lib/index.js??clonedRuleSet-1[0].rules[0]!./node_modules/vue-loader/lib/index.js??vue-loader-options!./src/App.vue?vue&type=script&lang=js& 15:0-48 20:15-24
 @ ./src/App.vue?vue&type=script&lang=js& 1:0-186 1:202-205 1:207-390 1:207-390
 @ ./src/App.vue 2:0-55 3:0-50 3:0-50 10:2-8
 @ ./src/main.js 29:0-28 85:13-16
```

[Full build log](https://pastebin.com/YHS1GdFP)

This can be solved by updating the node dependencies:

```sh
cd  web/server/vue-cli
npm update
```

**CodeChecker version**

6.22.2

**To Reproduce**

Steps to reproduce the behaviour:

On a system, or in a container, with a recent Node.js, follow the build instructions.

**Expected behaviour**

CodeChecker is buildable with recent node versions.

**Desktop (please complete the following information)**

 - OS: Arch Linux
 - Browser: N/A
 - Version: N/A

**Comment by bruntib** [comment]
Hi,
Thanks for maintaining the CodeChecker package in Arch linux!
I could reproduce the error messages. We have upgraded some 3pp dependencies and now the packaging works on my machine. Could you check, if the problem is solved?
Thank you!

**Comment by carlsmedstad** [comment]
I've confirmed it is fixed in master (f912cf0ccc7059240ae389823faf35225e6cabc8), thanks!

**Comment by bruntib** [comment]
Great news, thanks for the feedback! I close this ticket in this case.
